lib/danger/request_source/bitbucket_server_api.rb

```diff
@@ -0,0 +1,70 @@
+# coding: utf-8
+require "danger/helpers/comments_helper"
+
+module Danger
+  module RequestSources
+    class BitbucketServerAPI
```

Comment:
Ah yeah, I wasn't looking at the full diff, @k0nserv is right, but that's a nitpick, I'm god with this getting merged into another branch so we can do a documentation run 👍